### PR TITLE
Upgrading to Mantle testkit 1.0 and PHPUnit 11

### LINKED ISSUE #26: Upgrade to Mantle Testkit v1.0
### Description

Now that Mantle Testkit has a v1 release, we should update from the 0.12.x release to the 1.x release.

### Use Case

We should keep our dependencies up-to-date.

### PR DESCRIPTION
**Pull request by srtfisher**
Converted with https://github.com/alleyinteractive/wp-to-psr-4 Resolves #26

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,14 +1,14 @@
+<?xml version="1.0"?>
 <phpunit
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (renatonascalves) [inline phpunit.xml]: is this aligned correctly?

File: tests/TestCsvBulkTask.php [hunk 2]
```diff
@@ -7,6 +7,8 @@
 
 declare( strict_types=1 );
```
Comment (renatonascalves) [inline tests/TestCsvBulkTask.php]:
```suggestion
declare(strict_types=1);
```

File: tests/TestCursor.php [hunk 3]
```diff
@@ -7,6 +7,8 @@
 
 declare( strict_types=1 );
```
Comment (renatonascalves) [inline tests/TestCursor.php]:
```suggestion
declare(strict_types=1);
```

File: tests/TestUserBulkTask.php [hunk 4]
```diff
@@ -7,16 +7,19 @@
 
 declare( strict_types=1 );
```
Comment (renatonascalves) [inline tests/TestUserBulkTask.php]:
```suggestion
declare(strict_types=1);
```

File: .github/workflows/all-pr-tests.yml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  # reduce the number of minutes used.
+  pr-tests:
+    # Don't run on draft PRs
+    if: github.event.pull_request.draft == false
+    # Timeout after 10 minutes
+    timeout-minutes: 10
+    # Define a matrix of PHP/WordPress versions to test against
+    strategy:
+      matrix:
+        php: [8.1, 8.2, 8.3]
```
Comment (renatonascalves) [inline .github/workflows/all-pr-tests.yml:20]: No support for PHP 8.0 anymore or just not testing it anymore?

**Review by renatonascalves — CHANGES_REQUESTED** [review]

File: .github/workflows/all-pr-tests.yml (hunk 5 above)
Comment (srtfisher) [reply .github/workflows/all-pr-tests.yml:20]: Testkit doesn't support it and it isn't a supported PHP version anymore. I don't see a need to keep supporting it IMO.

Comment (renatonascalves) [reply .github/workflows/all-pr-tests.yml:20]: @srtfisher Could you update this too then: https://github.com/alleyinteractive/wp-bulk-task?tab=readme-ov-file#install

**Comment by renatonascalves** [comment]
@srtfisher Could you also update the required github actions in the branch protections?

**Comment by srtfisher** [comment]
@renatonascalves ready for another pass

**Review by renatonascalves — APPROVED** [review]
🌵

**Pull request merged**